tests/queries/0_stateless/01881_join_on_conditions_merge.sql.j2

```diff
@@ -4,6 +4,8 @@ DROP TABLE IF EXISTS t2_nullable;
 DROP TABLE IF EXISTS t2_lc;
 DROP TABLE IF EXISTS t22;
 
+SET enable_analyzer = 0;
```

Comment:
@antaljanosbenjamin could you explain why the analyzer has been disabled here?

Reply:
Probably by mistake. I will check this once I am back from holidays. Hopefully we I didn't break anything.